tasks/managed/apply-mapping/README.md

```diff
@@ -25,11 +25,18 @@ You can also expand image labels, e.g. "{{ labels.mylabel }}" -> The value of im
 
 ## Parameters
 
-| Name              | Description                                                                                  | Optional | Default value |
-|-------------------|----------------------------------------------------------------------------------------------|----------|---------------|
-| snapshotPath      | Path to the JSON string of the Snapshot spec in the config workspace to apply the mapping to | No       | -             |
-| dataPath          | Path to the JSON string of the merged data to use in the data workspace                      | No       | -             |
-| failOnEmptyResult | Fail the task if the resulting snapshot contains zero components                             | Yes      | false         |
+| Name                    | Description                                                                                                                 | Optional | Default value |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| snapshotPath            | Path to the JSON string of the Snapshot spec in the config workspace to apply the mapping to                                | No       | -             |
+| dataPath                | Path to the JSON string of the merged data to use in the data workspace                                                     | No       | -             |
+| failOnEmptyResult       | Fail the task if the resulting snapshot contains zero components                                                            | Yes      | false         |
+| ociStorage              | The OCI repository where the Trusted Artifacts are stored.                                                                  | No       | -             |
```

Comment:
This and line 34 are the only ones with `.` at the end. Probably don't need it

Reply:
fixed